src/lib/optimizer/join_ordering/abstract_join_ordering_algorithm.cpp

```diff
@@ -102,7 +102,7 @@ std::shared_ptr<AbstractLQPNode> AbstractJoinOrderingAlgorithm::_add_join_to_pla
     lqp = JoinNode::make(JoinMode::Cross, left_lqp, right_lqp);
   }
 
-  // non operator join predicates have to be processed as scans.
+  // Post-JoinNode predicates are processed as scans.
```

Comment:
Isn't "scan" a PQP concept?

Reply:
> Isn't "scan" a PQP concept?

+1 and could you elaborate this a little bit? I don't think outsiders understand what the alternative is that you decided against.